.travis.yml

```diff
@@ -0,0 +1,57 @@
+language: android
+jdk: oraclejdk8
+sudo: true
+env:
+  matrix:
+  - ANDROID_TARGET=android-21 ANDROID_ABI=armeabi-v7a
+  global:
+  - ADB_INSTALL_TIMEOUT=10
+before_cache:
+- rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
+- rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+cache:
+  directories:
+  - "$HOME/.gradle/caches"
+  - "$HOME/.android/build-cache"
+  - "$HOME/.gradle/wrapper"
+android:
+  components:
+  - tools
+  - platform-tools
+  - tools
+  - build-tools-25.0.2
+  - android-25
```

Comment:
Yes, this is to target a SDK Platform Tools version,  https://docs.travis-ci.com/user/languages/android/#Installing-a-newer-SDK-Platform-Tools-revision